### PR TITLE
JBIDE-20319: Automatic project refresh after 'Bower Update' execution. Adding missing license headers

### PR DESCRIPTION
**Pull request by ibuziuk**

### CODE REVIEW

File: plugins/org.jboss.tools.jst.js/src/org/jboss/tools/jst/js/bower/internal/launch/shortcut/GenericNativeBowerLaunch.java [hunk 1]
```diff
@@ -62,28 +71,57 @@ public void launch(IEditorPart editor, String mode) {
...
+				if (event.getKind() == DebugEvent.TERMINATE) {
+					Object source = event.getSource();
+					if (source instanceof IProcess) {
+						ILaunch launch = ((IProcess) source).getLaunch();
+						if (launch != null) {
+							ILaunchConfiguration lc = launch.getLaunchConfiguration();
+							if (lc != null && getLaunchName().equals(lc.getName()) && workingProject != null
+									&& workingProject.exists()) {
+								try {
+									workingProject.refreshLocal(IResource.DEPTH_INFINITE, null);
```
Comment (alexeykazakov) [inline plugins/org.jboss.tools.jst.js/src/org/jboss/tools/jst/js/bower/internal/launch/shortcut/GenericNativeBowerLaunch.java]: Shouldn't we remove the listener after refreshing the project?

Comment (ibuziuk) [inline plugins/org.jboss.tools.jst.js/src/org/jboss/tools/jst/js/bower/internal/launch/shortcut/GenericNativeBowerLaunch.java]: @alexeykazakov good catch ;-)

**Comment by ibuziuk** [comment]
PR was updated

File: plugins/org.jboss.tools.jst.js/src/org/jboss/tools/jst/js/bower/internal/launch/shortcut/GenericNativeBowerLaunch.java [hunk 2]
```diff
@@ -62,28 +71,58 @@ public void launch(IEditorPart editor, String mode) {
...
+									DebugPlugin.getDefault().removeDebugEventListener(this);
```
Comment (alexeykazakov) [inline plugins/org.jboss.tools.jst.js/src/org/jboss/tools/jst/js/bower/internal/launch/shortcut/GenericNativeBowerLaunch.java]: I would put this removing into the finally{...} block of this try-catch.

**Comment by ibuziuk** [comment]
Another good catch. PR was updated

**Pull request merged**